test/hotspot/gtest/gc/shared/test_oopStorageSet.cpp

```diff
@@ -140,15 +155,15 @@ TEST_VM_F(OopStorageSetTest, print_containing) {
   // Goofy values print nothing: unaligned out of storage pointer.
   {
     stringStream ss;
-    bool printed = OopStorageSet::print_containing((oop*)0x1, &ss);
+    bool printed = OopStorageSet::print_containing((char*)0x1, &ss);
     ASSERT_FALSE(printed);
     EXPECT_STREQ("", ss.freeze());
   }
 
   // Goofy values print nothing: aligned out of storage pointer.
   {
     stringStream ss;
-    bool printed = OopStorageSet::print_containing((oop*)0x8, &ss);
+    bool printed = OopStorageSet::print_containing((char*)0x8, &ss);
```

Comment:
Instead of hard-coded 0x8, instead use alignof(oop).

Reply:
Right. Fixed.